vignettes/reuse.Rmd

```diff
@@ -35,6 +35,53 @@ You can document multiple functions in the same file by using either `@rdname` o
 It's a technique best used with care: documenting too many functions in one place leads to confusion.
 Use it when all functions have the same (or very similar) arguments.
 
+### `@rdname`
+
+Use `@rdname <destination>` to include multiple functions in the same page.
+The functions will be collected and share the same `@title`, `@description`, `@examples`, etc.
+
+There are two basic ways to use it.
+You can create a dummy documentation block and then add the functions to it.
```

Comment:
nit: recently have been avoiding the term 'dummy' (e.g. https://blog.leeandlow.com/2012/08/16/when-did-the-word-dummy-become-derogatory/ / https://source.android.com/docs/setup/contribute/respectful-code#term-examples / https://nvlpubs.nist.gov/nistpubs/ir/2021/NIST.IR.8366.pdf), usually 'placeholder' serves the purpose equally well.

Reply:
Good point. And I think "standalone" is more clear here anyway.